projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -30,7 +30,6 @@ public function reregister_menu_items() {
 		$this->add_page_menu();
 		$this->add_testimonials_menu();
 		$this->add_portfolio_menu();
-		$this->add_comments_menu();
```

Comment:
Is this deletion intentional? It doesn't seem related to themes.

Reply:
Definitely not, thanks for noticing 🤦

Reply:
Updated now 😉